python/ray/data/dataset.py

```diff
@@ -3647,8 +3660,8 @@ def to_modin(self) -> "modin.DataFrame":
         Time complexity: O(dataset size / parallelism)
```

Comment:
We don't have a `Dataset.to_arrow` method

Reply:
this must be referring to `Dataset.to_arrow_refs()`, will update